rcl/include/rcl/time.h

```diff
@@ -123,6 +141,102 @@ RCL_WARN_UNUSED
 rcl_ret_t
 rcl_steady_time_point_now(rcl_steady_time_point_t * now);
 
+/// Retrieve the current time as a rcl_ros_time_point_t struct.
+/* This function returns the time from a ros clock unless simulation_time
+ * is active. In which case it returns the latest simulated time.
+ * The closest equivalent would be to std::chrono::steady_clock::now();
```

Comment:
Few things, first should it say it returns time from a system clock and not a ros clock? Second, wouldn't the closest equivalent be a system_clock?